frontend/packages/topology/integration-tests/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "promise/catch-or-return": "off",
+    "promise/no-nesting": "off"
```

Comment:
why we need this?

Reply:
This is used for a unit test case

`cy.readFile(yamlLocation).then((str) => {` in reqource-quota-warning.ts is not a promise, but eslint was throwing error messages for not having return and catch and also can't call nested promise. This is already added in other plugins, for toplogy it was not there, I added it.